post-install/post_setup

```diff
@@ -100,19 +100,24 @@
   rm -Rf /etc/sddm.conf
 
   # Installing yay helper
-  #git clone https://aur.archlinux.org/yay.git && cd yay && makepkg -si --noconfirm
+  git clone https://aur.archlinux.org/yay.git && cd yay && makepkg -si --noconfirm
 
   # Installing plymouth
-  #yay -S plymouth --noconfirm
+  yay -S plymouth --noconfirm
 
   # Setting pearOS plymouth theme
-  #plymouth-set-default-theme -R pear-plymouth
-
-
+  plymouth-set-default-theme -R pear-plymouth
+  
+  #Install flatpak
+  sudo pacman -S flatpak
```

Comment:
if you want to make flatpak installed by default you need to modify the initial pacstrap file (located in the github.com/pearOS-archlinux/iso/ )